### PR TITLE
chkconfig: add page

### PR DESCRIPTION
**Pull request by tunght13488**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/chkconfig.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# chkconfig
+
+> Manage service runlevel.
```
Comment (sbrl) [inline pages/linux/chkconfig.md]: I assume that this command is part of a service manager of some description? I've got ubuntu with systemd - and it's not installed.

Perhaps we could expand this description to include the service manager that this works with? For example `Manage insert_service_manager_name_here service runlevels.` would work?

Comment (tunght13488) [reply pages/linux/chkconfig.md]: @sbrl this is the service manager for Centos 6. I'm not sure about Red Hat/Fedora though as I'm not using any of them.

For latest Centos with systemd we can do something like `systemctl {enable|disable} <service_name>`. The equivalent on Centos 6 is `chkconfig <service_name> on`

Comment (sbrl) [reply pages/linux/chkconfig.md]: Ah, I see. Perhaps we could reword this to `Manage the runlevel of services on CentOS 6`?

Comment (tunght13488) [reply pages/linux/chkconfig.md]: @sbrl sounds good to me. Updated

**Comment by sbrl** [comment]
Thanks, @tunght13488 :smiley_cat:
